cmd/dp-observation-extractor/main.go

```diff
@@ -3,157 +3,248 @@ package main
 import (
```

Comment:
In some repo's we have moved the main.go file into the root of the repo. Is that something we are doing as part of these PR's?

Reply:
I think we didn't do it for other services either, but it would be nice to have a consistent main.go location.